files/style.css

```diff
@@ -0,0 +1,175 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background: #1d2379;
+  position: absolute;
+  background-image: url("Ellipse 18.png");
+  background-repeat: no-repeat;
+}
+
+.heading {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+}
+
+.logo {
+  position: relative;
+  align-self: flex-start;
+  color: #fff;
+  font-weight: 600;
```

Comment:
- Please properly align the header logo and the hamburger menu as it is not aligned like that of the figma design.

##### Your design
![Screenshot from 2022-07-01 19-22-06](https://user-images.githubusercontent.com/58771507/176951771-8037a046-e54f-48b8-a2de-67b95f8b0d30.png)

##### Figma design
![Screenshot from 2022-07-01 19-32-03](https://user-images.githubusercontent.com/58771507/176951845-5f8cf163-e1af-43b9-9da4-494aa5abb4c1.png)